### PR TITLE
Add SQL CLI deployment instructions

### PR DESCRIPTION
**Pull request by tatiana**
This feature is related to the SQL CLI 0.3 available in the Astro CLI 1.10.
This should only be published once #1763 is published.

@jwitz please, feel free to update this PR in place.

### CODE REVIEW

File: astro/cli/sql-cli.md [hunk 1]
```diff
@@ -105,7 +115,32 @@ connections:
...
+To store sensitive credentials in your `configuration.yml` files, such as API keys or user logins, create a file named `.env` in the root of your SQL project and define your credentials there. You can then reference these values in your `configuration.yml` files as environment variables. 
+
+For example, consider the following `.env` file:
+
+```text
+AWS_ACCESS_KEY_ID=<your-aws-access-key-id>
+AWS_SECRET_ACCESS_KEY=<your-aws-access-key-secret>
+```
+
+You can call these values in a `configuration.yml` file using the format `$ENVIRONMENT_VARIABLE_KEY`. For example:
```
Comment (jwitz) [inline astro/cli/sql-cli.md:127]:
```suggestion
You can call these values in a `configuration.yml` file using the format `$ENVIRONMENT_VARIABLE_KEY`.
```

File: astro/cli/sql-cli.md [hunk 2]
```diff
@@ -284,3 +328,65 @@ You can now run the SQL workflow as a DAG from your Astro project. See [Build an
 If you configured databases in your SQL CLI project, you must manually reconfigure these databases in your Astro project through the Airflow UI. See [Manage connections in Apache Airflow](https://docs.astronomer.io/learn/connections).
 
 :::
+
+
+### Deploy a SQL workflow to Astro Cloud
```
Comment (jwitz) [inline astro/cli/sql-cli.md:333]:
```suggestion
### Deploy a SQL workflow to Astro
```

File: astro/cli/sql-cli.md [hunk 3]
```diff
@@ -284,3 +328,65 @@ You can now run the SQL workflow as a DAG from your Astro project. See [Build an
...
+
+```
+astro login
+```
+
+Once those two conditions are met, it is possible to deploy a SQL workflow to Astro Cloud by using the command `astro flow deploy`:
+
+```sh
+astro flow deploy <workflow-name> --env <env>
+```
```
Comment (jwitz) [inline astro/cli/sql-cli.md:352]:
```suggestion
1. Run the following command to create an Astro project at the root of your SQL project:

    ```sh
    astro dev init
    ```

2. Run the following command to log in to Astro:

    ```sh
    astro login
    ```

3.  Run the following command to deploy a SQL workflow to Astro:

    ```sh
    astro flow deploy <workflow-name> --env <env>
    ```

```

File: astro/cli/sql-cli.md [hunk 4]
```diff
@@ -284,3 +328,65 @@ You can now run the SQL workflow as a DAG from your Astro project. See [Build an
...
+
+and
+
+```sh
+Which Astro Cloud deployment should be associated withdefault?
+ #     DEPLOYMENT NAME                 RELEASE NAME                 DEPLOYMENT ID                   
+ 1     customer-ds-dev                 universal-wave-2402          cl8bqua474573873jwenjhb6pab     
+ 2     customer-ds-stage               galactian-meteoroid-2760     clczsjio0134626cs2uy2xd9ads 
+```
```
Comment (jwitz) [inline astro/cli/sql-cli.md:376]:
```suggestion
5. Follow the CLI instructions to select a Workspace and Deployment for your SQL environment. Now when you run `astro flow deploy <workflow-name> --env <env>`, the CLI automatically deploys your workflow to the Deployment associated with your SQL environment.
```

File: astro/cli/sql-cli.md [hunk 5]
```diff
@@ -284,3 +328,65 @@ You can now run the SQL workflow as a DAG from your Astro project. See [Build an
...
+ Airflow UI: astronomer.astronomer.run/d0fcz47b?orgId=org_0FGQMRjnZdcx3EDk
+```
+
+From this moment, the user will be able to see/interact with the SQL workflows using the Airflow UI in Astro Cloud.
+
+:::warning 
+
+At the moment, the user is expected to configure the connections in the Astro deployment. There are plans to automate this step in the roadmap.
+
+:::
```
Comment (jwitz) [inline astro/cli/sql-cli.md:392]:
```suggestion

7. Reconfigure any connections from your local SQL project in the Airflow UI for your Deployment. See [Connections](https://docs.astronomer.io/learn/connections).
```

**Review by jwitz — CHANGES_REQUESTED** [review]
@tatiana Thank you for submitting this PR! I left some suggestions where I made assumptions about the CLI workflow, please review my comments and let me know if I'm headed in the right direction with this.

File: astro/cli/sql-cli.md (hunk 4 above)
Comment (tatiana) [reply astro/cli/sql-cli.md:363]: Well spotted, @jwitz , I'll add this

File: astro/cli/sql-cli.md [hunk 6]
```diff
@@ -105,7 +115,32 @@ connections:
...
+To store sensitive credentials in your `configuration.yml` files, such as API keys or user logins, create a file named `.env` in the root of your SQL project and define your credentials there. You can then reference these values in your `configuration.yml` files as environment variables. 
+
+For example, consider the following `.env` file:
+
+```text
+AWS_ACCESS_KEY_ID=<your-aws-access-key-id>
+AWS_SECRET_ACCESS_KEY=<your-aws-access-key-secret>
+```
+
+You can call these values in a `configuration.yml` file using the format `$ENVIRONMENT_VARIABLE_KEY`.
```
Comment (jwitz) [inline astro/cli/sql-cli.md:127]:
```suggestion
You can call these values in a `configuration.yml` file using the format `$ENVIRONMENT_VARIABLE_KEY`. For example: 
```

File: astro/cli/sql-cli.md [hunk 7]
```diff
@@ -284,3 +319,62 @@ You can now run the SQL workflow as a DAG from your Astro project. See [Build an
...
+    astro dev init
+    ```
+
+2. Run the following command to log in to Astro:
+
+    ```sh
+    astro login
+    ```
+
+3. (optional) Create the `workflow.yml` inside the desired workflow folder including scheduling details.
```
Comment (jwitz) [inline astro/cli/sql-cli.md:340]:
```suggestion
3. Optional. In your workflow directory, create a file named `workflow.yml` that includes configurations for scheduling your SQL workflow. For example:
```

File: astro/cli/sql-cli.md [hunk 8]
```diff
@@ -284,3 +319,62 @@ You can now run the SQL workflow as a DAG from your Astro project. See [Build an
...
+
+Example:
```
Comment (jwitz) [inline astro/cli/sql-cli.md:342]:
```suggestion
```

File: astro/cli/sql-cli.md [hunk 9]
```diff
@@ -284,3 +319,62 @@ You can now run the SQL workflow as a DAG from your Astro project. See [Build an
...
+      is_paused_upon_creation: false
+      tags:
+        - AWS
+        - S3
+        - Snowflake
+    ```
+
+It is also possible to have schedules triggered by other datasets, by using:
+
+    ```yaml
```
Comment (jwitz) [inline astro/cli/sql-cli.md:359]: Is this using Airflow datasets? Is the resulting DAG waiting for these datasets to be updated, or is it updating the datasets itself?

File: astro/cli/sql-cli.md [hunk 10]
```diff
@@ -284,3 +319,62 @@ You can now run the SQL workflow as a DAG from your Astro project. See [Build an
...
+
+
+4. Run the following command to deploy a SQL workflow to Astro:
+
+    ```sh
+    astro flow deploy <workflow-name> --env <env>
+    ```
+
+5. If you haven't yet installed the Astro Python SDK in your Astro project, the CLI prompts you to update `requirements.txt` in your Astro project. Enter `yes` to have the CLI automatically install the dependency.
+5. Follow the CLI instructions to select a Workspace and Deployment for your SQL environment. Now when you run `astro flow deploy <workflow-name> --env <env>`, the CLI automatically deploys your workflow to the Deployment associated with your SQL environment.
```
Comment (jwitz) [inline astro/cli/sql-cli.md:376]:
```suggestion
    If you haven't yet installed the Astro Python SDK in your Astro project, the CLI prompts you to update `requirements.txt` in your Astro project. Enter `yes` to have the CLI automatically install the dependency.
   
5. Follow the CLI instructions to select a Workspace and Deployment for your SQL environment. Now when you run `astro flow deploy <workflow-name> --env <env>`, the CLI automatically deploys your workflow to the Deployment associated with your SQL environment.
```

**Review by jwitz — COMMENTED** [review]
@tatiana One question about the datasets feature, otherwise this looks good to me!

File: astro/cli/sql-cli.md [hunk 11]
```diff
@@ -284,3 +318,67 @@ You can now run the SQL workflow as a DAG from your Astro project. See [Build an
...
+    astro dev init
+    ```
+
+2. Run the following command to log in to Astro:
+
+    ```sh
+    astro login
+    ```
+
+3. Optional. In your workflow folder, create a file named `workflow.yml` that defines when your workflow should run. You can also use this file to define metadata for the generated DAG. For example, the following configuration runs a SQL workflow hourly and will backfill all workflow runs until December 2021.
```
Comment (jwitz) [inline astro/cli/sql-cli.md:342]:
```suggestion
3. Optional. In your workflow folder, create a file named `workflow.yml` that defines when your workflow should run. You can also use this file to define metadata for the generated DAG. For example, the following configuration runs a SQL workflow hourly and backfills all workflow runs until December 2021. 
```

**Review by jwitz — APPROVED** [review]

**Pull request merged**